cmd/backup-operator/main.go

```diff
@@ -74,5 +76,9 @@ func createRecorder(kubecli kubernetes.Interface, name, namespace string) record
 }
 
 func run(stop <-chan struct{}) {
-	// TODO
+	c := controller.New()
+	err := c.Start(context.TODO())
+	if err != nil {
+		logrus.Infof("operator stopped with %v", err)
```

Comment:
What was the case that it would return error again?

Reply:
the init() function is not yet implemented

```go
func (b *Backup) init(ctx context.Context) error {
	// TODO: check if crd exists
	return nil
}
```

it can return errors.